scalalib/src/mill/scalalib/TestModule.scala

```diff
@@ -1,13 +1,22 @@
 package mill.scalalib
 
 import mill.{Agg, T}
-import mill.define.{Command, Task, TaskModule}
-import mill.api.{Ctx, Result}
+import mill.define.{Command, ModuleRef, Task, TaskModule}
+import mill.api.{Ctx, PathRef, Result}
 import mill.util.Jvm
 import mill.scalalib.bsp.{BspBuildTarget, BspModule}
 import mill.testrunner.TestRunner
 
-trait TestModule extends JavaModule with TaskModule {
+trait TestModule extends TaskModule with TestModule.JavaModuleBase {
```

Comment:
Why was this necessary? Now, that `TestModule` is no lomger a complete `JavaModule`, my `mill-jacoco` plugin broke.

Reply:
Turns out just mix-in `JavaModule` to my `JacacoTestModule` fixed the compilation in my case.